src/basic/FStar.Options.fs

```diff
@@ -1431,6 +1438,14 @@ let hint_file                    () = get_hint_file                   ()
 let ide                          () = get_ide                         ()
 let print                        () = get_print                       ()
 let print_in_place               () = get_print_in_place              ()
+let profile (f:unit -> 'a) (msg:'a -> string) : 'a =
```

Comment:
We could print the profiling message like `Starting task <string> ... Ending task <string>, total time taken: ...`. That way the nesting of  `Options.profile` calls will be clearer in the output.

But it is minor, we can do this later as we try to use `Options.profile` more.